dev/devicelab/lib/tasks/integration_tests.dart

```diff
@@ -139,6 +139,48 @@ TaskFunction createSolidColorTest({required bool enableImpeller}) {
   ).call;
 }
 
+// Can run on emulator or physical android device.
+// Device must have developer settings enabled.
+// Device must be android api 30 or higher.
+TaskFunction createDisplayCutoutTest() {
+  return IntegrationTest(
+    '${flutterDirectory.path}/dev/integration_tests/display_cutout_rotation/',
+    'integration_test/display_cutout_test.dart',
+    setup: (Device device) async {
+      if (device is AndroidDevice) {
+        // Test requires developer settings added in 28 and behavior added in 30.
+        final String sdkResult = await device.shellEval('getprop', <String>[
+          'ro.build.version.sdk',
+        ]);
+        if (sdkResult.startsWith('2') || sdkResult.startsWith('1') || sdkResult.length == 1) {
+          print('This test must be run on api 30 or higher.');
+          throw TaskResult.failure('This test should only target android 30+.');
```

Comment:
Doesn't seem necessary to print the same message that will be thrown one line lower.

Reply:
removed